db/index_test.go

```diff
@@ -324,28 +304,6 @@ func TestCreateIndex_IfFieldHasNoDirection_DefaultToAsc(t *testing.T) {
 	assert.Equal(t, client.Ascending, newDesc.Fields[0].Direction)
 }
 
-func TestCreateIndex_IfNameIsNotSpecified_Generate(t *testing.T) {
```

Comment:
question: Why did you remove this test?

Reply:
This is documented in the commit message:

> One test was removed, TestCreateIndex_IfNameIsNotSpecified_Generate - this is because it only pretends to test what it tests - the name _is_ set later by the unit test framework, and as such is identical to other tests here.